Example Apps/Example Swift/MenuManager.swift

```diff
@@ -113,6 +114,40 @@ private extension MenuManager {
             VehicleDataManager.checkPhoneCallCapability(manager: manager, phoneNumber:"555-555-5555")
         })
     }
+    
+    /// Menu item that changes the default template
+    ///
+    /// - Parameter manager: The SDL Manager
+    /// - Returns: A SDLMenuCell object
+    class func menuCellChangeTemplate(with manager: SDLManager) -> SDLMenuCell {
+    
+        /// Lets give an example of 2 templates
+        var submenuItems = [SDLMenuCell]()
+        
+        /// Non-Media
+        let submenuTitleNonMedia = "Non - Media (Default)"
+        submenuItems.append(SDLMenuCell(title: submenuTitleNonMedia, icon: nil, voiceCommands: nil, handler: { (triggerSource) in
+            let display = SDLSetDisplayLayout(predefinedLayout: .nonMedia)
+            manager.send(request: display) { (request, response, error) in
+                if response?.resultCode == .success {
```

Comment:
Add a guard statement that shows an alert if changing the template fails. There is an example in one of the other menu items.